fetch.bs

```diff
@@ -6160,6 +6186,14 @@ method steps are to <a for=Headers>append</a> (<var>name</var>, <var>value</var>
  <a for=Headers>header list</a>.
 </ol>
 
+<p>The <dfn export for=Headers method><code>getSetCookie()</code></dfn> method steps are:
+
+<ol>
+ <li><p>Return <a lt=value for=header>values</a> of all <a for=/>headers</a> in <a>this</a>'s
+ <a for=Headers>header list</a> whose <a for=header>name</a> is a <a>byte-case-insensitive</a> match
+ for `<code>set-cookie</code>`, in order.
+</ol>
```

Comment:
This needs a step that returns an empty list when there are no such headers.

The second step should say "Return the values ... Set-Cookie ..." (in particular "the" and casing).

Reply:
> This needs a step that returns an empty list when there are no such headers.

Why? `[].filter((header) => header.name === "set-cookie").length === 0`. Does filtering on lists in specs not return an empty list for no matches?

Reply:
That's an interesting idea, but I'm not sure from what it would follow. The way I read the prose is that it's not accounted for what happens when there is nothing named Set-Cookie.